src/Compilers/Core/Portable/project.json

```diff
@@ -1,8 +1,26 @@
 ﻿{
   "supports": {},
   "dependencies": {
+    "System.Collections": "4.0.11",
```

Comment:
Most of these should come from `NETStandard.Library` and `Microsoft.NETCore.Portable.Compatibility`